helicone-node/async_logger/HeliconeAsyncLogger.ts

```diff
@@ -1,125 +1,99 @@
-import { IHeliconeAsyncClientOptions } from "../core/HeliconeClientOptions";
-import { fetch, Response } from "@whatwg-node/fetch";
+import * as traceloop from "@traceloop/node-server-sdk";
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
 
-export type HeliconeAsyncLogRequest = {
-  providerRequest: ProviderRequest;
-  providerResponse: ProviderResponse;
-  timing: Timing;
-};
-
-export type ProviderRequest = {
-  url: string;
-  json: {
-    [key: string]: any;
-  };
-  meta: Record<string, string>;
-};
-
-export type ProviderResponse = {
-  json: {
-    [key: string]: any;
-  };
-  status: number;
-  headers: Record<string, string>;
-};
+import OpenAI from "openai";
+import * as anthropic from "@anthropic-ai/sdk";
+import * as azureOpenAI from "@azure/openai";
+import * as cohere from "cohere-ai";
+import * as bedrock from "@aws-sdk/client-bedrock-runtime";
+import * as google_aiplatform from "@google-cloud/aiplatform";
+import * as pinecone from "@pinecone-database/pinecone";
+import * as ChainsModule from "langchain/chains";
+import * as AgentsModule from "langchain/agents";
+import * as ToolsModule from "langchain/tools";
+import * as chromadb from "chromadb";
 
-export type Timing = {
-  // From Unix epoch in Milliseconds
-  startTime: {
-    seconds: number;
-    milliseconds: number;
-  };
-  endTime: {
-    seconds: number;
-    milliseconds: number;
-  };
+type IHeliconeAsyncLoggerOptions = {
+  apiKey: string;
+  baseUrl: string;
+  providers: {
+    openAI?: typeof OpenAI,
+    anthropic?: typeof anthropic,
+    azureOpenAI?: typeof azureOpenAI,
+    cohere?: typeof cohere,
+    bedrock?: typeof bedrock,
+    google_aiplatform?: typeof google_aiplatform,
+    pinecone?: typeof pinecone,
+    langchain?: {
+      chainsModule?: typeof ChainsModule,
+      agentsModule?: typeof AgentsModule,
+      toolsModule?: typeof ToolsModule,
+    },
+    chromadb?: typeof chromadb,
+  },
+  headers?: Partial<Record<string, unknown>>
```

Comment:
Partial is redundant, and headers should always be a string
```suggestion
  headers?: Record<string, string>
```